Gemfile.lock

```diff
@@ -188,6 +189,7 @@ GEM
       slop (~> 3.4)
     pry-rails (0.3.4)
       pry (>= 0.9.10)
+    public_suffix (2.0.5)
```

Comment:
@BenMorganIO `public_suffix` gem is the dependency for addressable gem. Please refer https://github.com/thoughtbot/administrate/pull/822/files#diff-e79a60dc6b85309ae70a6ea8261eaf95R55 . Do we still want to remove it?